server/multitenancy/tenant_resolver.ts

```diff
@@ -46,10 +46,14 @@ export function resolveTenant(
   let selectedTenant: string | undefined;
   const securityTenant_ = request?.url?.searchParams?.get('securityTenant_');
   const securitytenant = request?.url?.searchParams?.get('securitytenant');
+  // eslint-disable-next-line @typescript-eslint/naming-convention
+  const security_tenant = request?.url?.searchParams?.get('security_tenant');
```

Comment:
What do you think about addressing the source of this parameter and pull the value from a shared constant?

https://github.com/opensearch-project/security-dashboards-plugin/blob/e08dca4eca8906ade344a00ca2b80289037522c3/public/services/shared-link.ts#L72-L77

https://github.com/opensearch-project/security-dashboards-plugin/blob/e08dca4eca8906ade344a00ca2b80289037522c3/public/apps/account/account-app.tsx#L31-L36

Reply:
I agree that we should stick to one convention for the naming of the url param for the tenant. 

The question I would have is if we want to support existing shared-links that have already been generated. If that's the case then this is a good fix, but we should understand why we are supporting 3 different permutations for the same url param.

Reply:
@peternied That's a good point! That can be done separately, but this fix is necessary for the exiting versions where different naming conventions exist.

Reply:
I would prefer we didn't support the old variants to keep the code paths simpler, they never worked correctly so an old those links around won't be any more negatively impacted

Reply:
@peternied Thanks for the good idea! I agree that having consistent naming conventions is ideal, but it would break the compatibility with existing shared-links in this case.
@jimishs confirmed that users actually run into the scenario where users bookmarked existing shared-links and got 404 when using the shared-links to visit the saved objects.